docs/IgorFromUA/calculator.html

```diff
@@ -0,0 +1,113 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="styles/styles.css">
+        <link rel="stylesheet" type="text/css" href="styles/header.css">
+        <link rel="stylesheet" type="text/css" href="styles/calculator.css">
+        <link rel="stylesheet" type="text/css" href="styles/footer.css">
+        <title>Calculator</title>
+    </head>
+    <body class="wrapper-all">
+        <div class="wrapper">
+            <header class="header container">
```

Comment:
Hey, @IgorFromUA, don't you like to remove these empty gaps?
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/e5b8d5f2-2f8a-48f9-ba47-e5b71fe63a7e)